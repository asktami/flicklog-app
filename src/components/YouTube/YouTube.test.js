import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import YouTube from './YouTube';

describe(`YouTube component`, () => {
	it('renders a YouTube by default', () => {
		const wrapper = shallow(<YouTube />);
		expect(toJson(wrapper)).toMatchSnapshot();
	});
});
