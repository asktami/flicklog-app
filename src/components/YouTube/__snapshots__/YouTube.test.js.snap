// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`YouTube component renders a YouTube video by default 1`] = `
<div
  className="video"
>
  <iframe
    frameBorder="0"
    src="https://www.youtube.com/embed/undefined"
  />
</div>
`;
